components/phpunit_bridge.rst

```diff
@@ -29,7 +29,7 @@ Installation
 
 .. code-block:: terminal
 
-    $ composer require --dev "symfony/phpunit-bridge:*"
+    $ composer require --dev "symfony/phpunit-bridge::^3.4"
```

Comment:
No double colon here and the quotes are not needed, right?

Reply:
Quotes are needed by some shells like zsh IIRC.

Reply:
Ok, but for now we are not using quotes across the docs, maybe we should add them everywhere then

Reply:
I think so. I confirm the behavior. Quoting is what I'm doing in Twig's docs: https://twig.symfony.com/doc/2.x/installation.html